Frameworks/Foundation/NSOrderedSet.mm

```diff
@@ -16,44 +16,41 @@
 
 #include "Starboard.h"
 #import <StubReturn.h>
+#include <vector>
 
-#include "Foundation/NSOrderedSet.h"
+#import <Foundation/NSOrderedSet.h>
+#import <Foundation/NSException.h>
+
+@implementation NSOrderedSet {
+    std::vector<StrongId<id>> setContainer;
```

Comment:
Note that "normal" ordered maps etc won't work for the ordered portion as it is not an ordering defined by hash value, or item comparison. IIRC, NSOrderedSet provides the ability to insert an item at a desired index.

---
In reply to: [61937816](https://github.com/Microsoft/WinObjC/pull/472#discussion_r61937816) [](ancestors = 61937816,61937416,61937213)